src/rebuild/rpc.h

```diff
@@ -56,7 +57,6 @@ extern struct crt_proto_format rebuild_proto_fmt;
 	((uint32_t)		(rsi_layout_ver)	CRT_VAR)
 
 #define DAOS_OSEQ_REBUILD_SCAN	/* output fields */		 \
-	((uint64_t)		(rso_stable_epoch)	CRT_VAR) \
```

Comment:
Since the rebuild RPC protocol version (DAOS_REBUILD_VERSION 4) is not being updated perhaps it is safest to keep the uint64_t member in this definition, but rename it to rso_reserved or something like that?

Reply:
as server rolling upgrade does not supported now, the DAOS_REBUILD_VERSION is not really make sense now. 
But I'll add a u64 reservation if need to refresh the PR later.